include/onnxruntime/core/graph/graph.h

```diff
@@ -201,14 +201,14 @@ class Node {
   // Functions defined to traverse a Graph as below.
 
   /** Gets an iterator to the beginning of the input nodes to this Node. */
-  NodeConstIterator InputNodesBegin() const noexcept { return NodeConstIterator(relationships_.input_edges.cbegin()); };
+  NodeConstIterator InputNodesBegin() const noexcept { return {relationships_.input_edges.cbegin()}; };
```

Comment:
This seems slightly more obtuse to interpret vs. explicitly constructing a NodeConstIterator. Is there a reason for doing it this way? I would have expected with RVO there wouldn't be a performance difference.

Reply:
Just for making it shorter. Hmm.. I'll revert this change back.